frontend/src/my_team/MyTeamView.jsx

```diff
@@ -44,7 +43,12 @@ function MyTeamView() {
     const queryUrl = new URL(`${PORT}/teamPlayers`);
     queryUrl.searchParams.append("userId", userContext.user.id);
     queryUrl.searchParams.append("teamType", teamType);
-    queryUrl.searchParams.append("playingStyle", style);
+
+    const sortStyle =
+      style === ""
+        ? recommendations.data.response.recommendedStyle.style
+        : style;
```

Comment:
https://github.com/user-attachments/assets/261a693c-3858-4603-a544-750cbaf04fae

Allows for automatically loading in the recommended roster. Because the scouting calculations identified insideOffense as the best playing style for the team, the menu is automatically set to that style.